recipes/pyaerocom/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyaerocom" %}
+{% set version = "0.8.0rc2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7822c778fb598e99a5e797751a49508385cda9c13c9b436c7a198280f550dbb0
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv "
```

Comment:
You need to define some `entry_points` to fix Windows: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points